### ISSUE TITLE
Max Camera Distance Not Saving

### ISSUE THREAD

**Issue by MetalHeadSeven** [Stanzilla/AdvancedInterfaceOptions#84]
I have "Enforce Settings on Startup" checked but Max Camera Distance resets every time I login or reload.

**Comment by xathian** [comment]
Yeah just noticed this on Remix last night, it keeps maxing out the value when I normally play with the default max.
